src/request/node.ts

```diff
@@ -62,6 +64,41 @@ export interface NodeRequestOptions<T> extends RequestOptions {
 	streamData?: boolean;
 	streamEncoding?: string;
 	streamTarget?: WritableStream<T>;
+	redirectOptions?: {
+		limit?: number;
+		count?: number;
+		keepOriginalMethod?: boolean;
+	};
+}
+
+function redirect<T>(resolve: (p?: any) => void, reject: (_?: Error) => void, url: string, options: NodeRequestOptions<T>): boolean {
+	if (!options.redirectOptions) {
+		options.redirectOptions = {};
+	}
+
+	const redirectLimit = options.redirectOptions.limit || defaultRedirectLimit,
+		{ count: redirectCount = 0 } = options.redirectOptions,
```

Comment:
[Style Guide, Variables Item 3](https://github.com/dojo/meta/blob/master/STYLE.md#variables) indicates that every variable should be declared on its own line with it own variable keyword, instead of using a list.